test/DatastoreTest.java

```diff
@@ -0,0 +1,12 @@
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
the file path should be the same as the class,
e.g DataStore is in java/com/google/codeu/data

its test should be test/com/google/codeu/data

can you write a